src/index.js

```diff
@@ -2,8 +2,24 @@
 
 import React from 'react';
 import { render } from 'react-dom';
+import * as Sentry from '@sentry/react';
+import { Integrations } from '@sentry/tracing';
+
 import App from './components/App';
 
 require('./favicon.ico'); // Tell webpack to load favicon.ico
 
+if (process.env.NODE_ENV === 'production' || true) {
+  Sentry.init({
+    dsn:
+      'https://ad5d766ba9fb46d784ee1418d837605b@o561173.ingest.sentry.io/5697727',
```

Comment:
this key should be fine to be public on github

Reply:
bad actors could send you fake errors but thats all

Reply:
ok as long as we have the option of regenerating the secret